profilegen/src/res/templates/spec_template.template

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require_relative 'lib/client'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'azure_mgmt_profile_<%= profile_version %>'
+  spec.version       = '<%= profile_ruby_version %>'
```

Comment:
Is this really a `profile_ruby_version`

Reply:
As discussed, please add VERSION file so that we can refer it here

Reply:
Done